client/reader/index.js

```diff
@@ -45,6 +45,8 @@ export async function lazyLoadDependencies() {
 }
 
 export default function() {
+	lazyLoadDependencies();
```

Comment:
Would still want to `await` here and declare the default `async`.  Note in `sections-middleware` that will `await` when invoking the default export for this reason:
https://github.com/Automattic/wp-calypso/pull/40913/commits/435782f1019de4cb736d68b997859718817dd64b#diff-afb3c8d4236d194988c31a97e465126dR39

Reply:
Right, this is still work in progress, I was testing if something like that helps since `loadSection` is not awaited.

Reply:
Sorry.  Didn't notice the label changes.

Reply:
No worries, I just pushed to see the ICFY stats, I'll have a final commit in a bit, I think you are right and it might work just fine even if the whole `loadSection` function is not awaited.

Reply:
Your suggestion worked like a charm, thanks